### ISSUE TITLE
Can't make requests: 'Issuer' claim ('iss') must be an Integer

### ISSUE THREAD

**Issue by awwright** [probot/friction#42]
I've run create-probot-app and setup a new GitHub Application, and I'm receiving and handling webhook events, but outgoing requests seem to be failing with Unauthorized errors. No information is provided on what's missing or incorrect.

I would guess Probot needs OAuth credentials to make outgoing requests (Client Id/Client secret), but I'm not sure where it accepts those.

```
> probot run ./index.js

01:09:09.257Z  INFO probot: Forwarding https://smee.io/... to http://localhost:3000/
01:09:09.263Z  INFO probot: Listening on http://localhost:3000
01:09:09.804Z  INFO probot: Connected https://smee.io/...
01:09:09.952Z DEBUG github: GitHub request: GET /app/installations - 401 undefined (installation=undefined)
  params: {
    "per_page": 100,
    "baseUrl": "https://api.github.com",
    "request": {
      "timeout": 0
    }
  }
01:09:09.953Z ERROR probot: {"message":"'Issuer' claim ('iss') must be an Integer","documentation_url":"https://developer.github.com/v3"}
  HttpError: {"message":"'Issuer' claim ('iss') must be an Integer","documentation_url":"https://developer.github.com/v3"}
      at response.text.then.message (.../github-bot/node_modules/@octokit/rest/lib/request/request.js:72:19)
      at processTicksAndRejections (internal/process/next_tick.js:81:5)
```

```
$ node -v
v11.10.0
$ ./node_modules/.bin/probot -V
7.5.0
```

**Comment by 2rojan** [comment]
I came across this issue yesterday, trying to sort out the same problem. I spent a ton of time digging deep into the Probot code base, trying to unravel the stack trace - looking for the solution to this issue. It turns out that, Probot is making a JWT token from the (in my case) GitHub App ID and the rsa key stored in the private-key.pem. In my case the issue was that the GitHub App ID - while directly in front of my face, was not easy to see in the tool - nothing to highlight it or draw attention to it, where as the Client ID (part of the OAuth business) that I was not interested in was easy to see - and I had used that by mistake. 
So, this happens when using a value that isn't the GitHub App ID - something that would make someone feel silly, because its there, but hard to see and 
That stupid ID needs something to draw more attention to it!
![GAI](https://user-images.githubusercontent.com/30205999/56984432-ff3a9e00-6b53-11e9-86fd-10bb3244478f.png)
